guides/deployment/docker.md

```diff
@@ -0,0 +1,304 @@
+# Introduction to Deployments with docker
+
+First things first, we need a running application to deploy it. If you don't yet have one, head to
+[Up and Running Guide](up_and_running.html).
+
+Docker enables you to deploy to the platform of your choice AWS, Google cloud, Digital ocean...
```

Comment:
I would add a semicolon after choice “...the platform of your choice: AWS, Google Cloud, Digital Ocean...” 

Note also the capital letters on the names.